uploadNeuroDB/minc_deletion.pl

```diff
@@ -1,4 +1,48 @@
 #! /usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+minc_deletion.pl -- this script deletes files records from the database, and
+deletes and archives the backend files stored in C</data/$PROJECT/data/assembly/>.
+Files to be deleted can be specified either based on the series UID or the file
+ID.
+
+=head1 SYNOPSIS
+
+perl minc_deletion.pl C<[options]>
+
+Available options are:
+
+-profile    : name of the config file in
+              C<../dicom-archive/.loris_mri>
+
+-series_uid : the series UID of the file to be deleted
+
+-fileid     : the file ID of the file to be deleted
+
+
+=head1 DESCRIPTION
+
+This program deletes MINC files from LORIS by:
+  - Moving the existing files (C<.mnc>, C<.nii>, C<.jpg>, C<.header>,
+    C<.raw_byte.gz>) to the archive directory: C</data/$PROJECT/data/archive/>
+  - Deleting all related data from C<parameter_file> & C<files> tables
+  - Deleting data from C<files_qcstatus> & C<feedback_mri_comments>
+    database tables if the C<-delqcdata> option is set. In most cases
+    you would want to delete this when the images change
+  - Deleting mri_acquisition_dates entry if it is the last file
```

Comment:
code format for table mri_acquisition_dates

Reply:
the last tiny change before I merge it :)

Reply:
done